week04/index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css">
+    
+    <title>가위바위보</title>
+</head>
+<body>
+    <div class="top_end"></div>
+    <div class="background">
+        <div class="top">
```

Comment:
class 이름은 목적과 의미가 드러나도록 명확하게 짓는 것이 좋습니다.
```suggestion
        <div class="result">
```